### PR TITLE
feature/fetch requests

### PR DESCRIPTION
**Pull request by DrewBradley**
### What does this PR do? (Fix/Feature/Bug)
Feature. Retrieves traveler, destination, and trips data from API to use for data model.

### Where should the reviewer start?
Begin with lines 1 - 77 of ```utilitiy.js``` and lines 20-30 of ```index.js```.

### How is this manually tested?
Run ```npm start``` in terminal and open application in browser on localhost:8080.  A random user's name should be appear. Arrays for Trips and Destinations should appear in the console.

### Any relevent tickets?
#10

### CODE REVIEW

File: src/Trips.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-    this.id = id;
-    this.userID = userID;
-    this.destinationID = destinationID;
-    this.travelers = travelers;
-    this.date = date;
-    this.duration = duration;
-    this.status = status;
-    this.suggestedActivities = suggestedActivities;
+  constructor(data) {
+    this.data = data;
```
Comment (mike-duke) [inline src/Trips.js:3]: i think it's fine here to pass in a single parameter (tripData would be more declarative), but i think you should still have the individual properties separated out, like "this.id = tripData.id, this.userID = tripData.userID". That way, you don't have to use "trip.data.id" or "trip.data.duration" every time you want to access a property.

File: src/index.js [hunk 2]
```diff
@@ -5,4 +5,28 @@ import './images/ExcursiOnward-logo.png'
...
+import {
+  getTraveler,
+  getTrips,
+  getDestinations,
+  addTrip,
+  updateTrip,
+  deleteTrip
+} from './utility.js';
+
+const dashboardGreeting = document.querySelector('.dashboard-greeting')
```
Comment (mike-duke) [inline src/index.js:19]: Do you think you will be changing the dashboard greeting enough to make it a global variable?

**Review by mike-duke — APPROVED** [review]
Overall, this looks good. You structure is solid and you're off to a great start!

File: src/index.js (hunk 2 above)
Comment (DrewBradley) [reply src/index.js:19]: I always forget that query selectors can live in the function that updates them. Thanks for the reminder!

File: src/Trips.js (hunk 1 above)
Comment (DrewBradley) [reply src/Trips.js:3]: Thanks. I think I avoid this because I was reprimanded in my Mod 1 assessment. I think it makes the most sense in this case.

**Pull request merged**